straxen/plugins/position_reconstruction.py

```diff
@@ -91,98 +109,84 @@ def compute(self, peaks):
             return result
 
         # Getting actual position reconstruction
-        _in = peaks['area_per_channel'][peak_mask, 0:self.config['n_top_pmts']]
+        area_per_channel_top = peaks['area_per_channel'][
+                               peak_mask,
+                               0:self.config['n_top_pmts']]
         with np.errstate(divide='ignore', invalid='ignore'):
-            _in = _in / np.max(_in, axis=1).reshape(-1, 1)
-        _in = _in.reshape(-1, self.config['n_top_pmts'])
-        _out = self.model.predict(_in)
+            area_per_channel_top = (
+                    area_per_channel_top /
+                    np.max(area_per_channel_top, axis=1).reshape(-1, 1)
+            )
+        area_per_channel_top = area_per_channel_top.reshape(-1,
+                                                            self.config['n_top_pmts']
+                                                            )

```

Comment:
FYI
```
area_per_channel_top = (area_per_channel_top.T / np.max(area_per_channel_top, axis=1)).T
```
works too and is a bit shorter. A bit like matrix multiplication, but not really :D